akka-actor/src/main/scala/akka/pattern/StatusReply.scala

```diff
@@ -47,6 +47,21 @@ final class StatusReply[+T] private (private val status: Try[T]) {
   def isError: Boolean = status.isFailure
   def isSuccess: Boolean = status.isSuccess
 
+  /**
+   * Return this StatusReply as a Try.
+   */
+  def toTry: Try[T] = status
```

Comment:
No problem with adding those. Normally StatusReply is supposed to be used with `askWithStatus` and the returned Future is already flattened to `Future[T]`. I guess you are using it without askWithStatus?